pkg/ociinstaller/imageref.go

```diff
@@ -87,11 +87,11 @@ func sanitizeRefStream(ref string) string {
 func (r *SteampipeImageRef) GetOrgNameAndStream() (string, string, string) {
 	// plugin.Name looks like `hub.steampipe.io/plugins/turbot/aws@latest`
 	split := strings.Split(r.DisplayImageRef(), "/")
-
-	org := split[len(split)-2]
 	pluginNameAndStream := strings.Split(split[len(split)-1], "@")
-
-	return org, pluginNameAndStream[0], pluginNameAndStream[1]
+	if strings.HasPrefix(r.DisplayImageRef(), "hub.steampipe.io/") {
```

Comment:
can we put a const in for `hub.steampipe.io` in the consts package

Reply:
Done!